docs/README.md

```diff
@@ -1,5 +1,128 @@
-# Documentation for the Splunk Distribution of OpenTelemetry Go
+# Unofficial documentation
```

Comment:
```suggestion
# Draft documentation
```

Reply:
@pellared We don't give a special name to GH docs in other repos, but it's interesting to think of one. I'd rather go with "Draft", as "Unofficial" gives the idea that this is not Splunk's effort, while it is. We do support these docs, too.